common4j/gradlew.bat

```diff
@@ -0,0 +1,89 @@
+@rem
```

Comment:
Does this need to be checked in? probably not

Reply:
Added to gitignore and force pushed to resolve this.